.azuredevops/deploy-platform-core.yaml

```diff
@@ -91,6 +91,19 @@ extends:
           - name: aks-cluster
             path: infra/managed-cluster
             resourceGroupName: $(aksResourceGroup)
+          - name: grafana
```

Comment:
can this go part of "monitoring" group ?

Reply:
or should it?

Reply:
@DaRoszko - I did initially put both Azure Monitor Workspace and Azure Grafana Dashboard in the Monitoring group, but after talking to Ajay we decided to put the Azure Monitor Workspace creation inside the AKS Bicep deployment, which meant we had to move the creation of Grafana after AKS.

Reply:
So in short it has to be deployed after the Azure Monitor Workspace which is part of the AKS Bicep deployment.

Reply:
thanks